storage/migrations/02_oasis_3_emerald.up.sql

```diff
@@ -204,6 +204,29 @@ CREATE TABLE oasis_3.emerald_withdraws
 CREATE INDEX ix_emerald_withdraws_sender ON oasis_3.emerald_withdraws(sender);
 CREATE INDEX ix_emerald_withdraws_receiver ON oasis_3.emerald_withdraws(receiver);
 
+-- Events emitted from the emerald paratime
+CREATE TABLE oasis_3.emerald_events
+(
+  round UINT63 NOT NULL REFERENCES oasis_3.emerald_rounds DEFERRABLE INITALLY DEFERRED,
+  tx_index UINT31,
+
+  tx_hash HEX64,
+  -- TODO: add link to openapi spec section with runtime event types.
+  type TEXT NOT NULL,
+  -- The raw event, as returned by the oasis-sdk runtime client. 
+  -- `evm.Log` events are further parsed into known event types, 
+  -- e.g. (ERC20) Transfer, to populate the `evm_log_name` and 
+  -- `evm_log_params` fields below.
+  body JSON NOT NULL,
+  evm_log_name TEXT,
+  evm_log_params JSONB,
+  related_accounts TEXT[],
```

Comment:
nit:
```suggestion
  related_accounts oasis_account[],
```
unless something goes wrong on the pgx side then